### DISCUSSION TITLE
How can I have a phantom camera follow behind a rigid body, and swing around as the rigid body rolls towards/away from the camera?

### DISCUSSION THREAD

**Discussion by azimuthdeveloper** [ramokz/phantom-camera#370] [Question / Help Wanted]
Basically, I have a ball that rolls. As the ball rolls, I want the camera to be behind the ball. But as the ball rolls around a corner and rolls in a new direction, the camera should slowly adjust its bearing and continue to follow the ball. If the ball were to stop or roll backwards, then the ball would pass below the camera (while the camera was still looking at it) and then continue to track the ball (but the camera would be facing the opposite direction to what it was at the outset.

I hope that makes sense... is this possible with phantom camera? I guess I'd have to use a swingarm or equivalent, but haven't been able to sort it out.

**Comment by ramokz** [comment]
It is possible, but there's no automatic or specific mode within the addon that does this, so you would need to do all the logical behaviour manually to achieve something like that.
